src/api/spec/cassettes/Staging_StagingProject/_copy/copies_the_repositories_and_it_s_relations.yml

```diff
@@ -0,0 +1,38 @@
+---
+http_interactions:
+- request:
+    method: put
+    uri: http://backend:5352/source/home:tom:Staging:C/_config?user=factory%20bot
+    body:
+      encoding: UTF-8
+      string: 'Prefer: foo'
+    headers:
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+      Accept:
+      - "*/*"
+      User-Agent:
+      - Ruby
+  response:
+    status:
+      code: 404
+      message: project 'home tom Staging C' does not exist
+    headers:
+      Content-Type:
+      - text/xml
+      Cache-Control:
+      - no-cache
+      Connection:
+      - close
+      Content-Length:
+      - '164'
+    body:
+      encoding: UTF-8
+      string: |
+        <status code="404">
+          <summary>project 'home:tom:Staging:C' does not exist</summary>
```

Comment:
globalwritethough must die

Reply:
there were enough other problems

Reply:
because there was one issue which is addressed in this PR... alright:wink: